static/app/views/discover/eventDetails/content.tsx

```diff
@@ -147,6 +152,10 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
     const eventJsonUrl = `/api/0/projects/${organization.slug}/${this.projectId}/events/${event.eventID}/json/`;
 
     const hasProfilingFeature = organization.features.includes('profiling');
+    const hasProfilingPreviewsFeature =
+      hasProfilingFeature && organization.features.includes('profiling-previews');
+
+    const profileId = isTransaction(event) ? event.contexts?.profile?.profile_id : null;
```

Comment:
Oh nice, I was just going to say that we should be careful not to delay anything on the page in case the flag is enabled, but we know user has sent no profiles. In that case, we'd want to avoid a loading state as we would know that it will fail and we'd just be impacting the UX, but  this mitigates it 👏🏼